packages/render-fidelity-tools/test/renderers/model-viewer/index.html

```diff
@@ -56,7 +56,12 @@
     harness.addEventListener('scenario-change', () => {
       const {scenario} = harness;
       modelViewer.addEventListener('poster-dismissed', () => {
-        this.dispatchEvent(new CustomEvent('model-ready'));
+        // When the poster-dismissed event fired, one raf has already been requrested. 
+        // Use an additional raf here is to make sure model-ready event fires at the frame that 
+        // the poster has completely faded away. 
+        requestAnimationFrame(()=>{
+          this.dispatchEvent(new CustomEvent('model-ready'));
```

Comment:
I think you'll need a raf inside a raf to get past the bug we found. A raf that dispatches an event that requests a raf can still call those two in the same animation frame.

Reply:
Thank you so much for pointing this out!